src/backend/base/langflow/services/telemetry/opentelemetry.py

```diff
@@ -64,13 +64,13 @@ def __init__(
         self,
         name: str,
         description: str,
-        type: MetricType,
+        metric_type: MetricType,
```

Comment:
Strictly speaking this is a breaking change. Is it acceptable here ?

Reply:
yes